lib/ansible/modules/windows/win_firewall_profile.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Simon Lecoq <simon.lecoq@live.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: win_firewall_profile
+version_added: "2.10"
+short_description: Windows firewall profile automation
+description:
+  - Allows you to update settings for Windows firewall profiles.
```

Comment:
Maybe mention it's the default action again

Reply:
Same here